### ISSUE TITLE
Merge slide text feature request

### ISSUE THREAD

**Issue by joelgraff** [ChurchApps/FreeShow#752]
When importing songs as a new show, Free Show will often over-segregate the slides, creating blank slides or slides with a few words at a time on them.  While I realize parsing lyrics from other sources can be troublesome, it would be nice to have a built-in mechanism to manage this.  It should perform at least two things:

1.  Automatically delete empty slides
2. Provide a merge option for slides.  Potential implementations:
    a. Create a "Merge" feature which merges selected slides into one
    b. Create "Merge Left / Right"  context menu action that will cause the text of the selected slide to be merged after the text in the slide to the left, or before the text in the slide to the right.
    
Implementation 2(b) seems most reasonable to me as the most common merge action is likely to address this issue of over-segregated lyric slides, and it has a well-defined behavior, as opposed to merging a random selection of slides.

**Comment by vassbo** [comment]
1. Using "Text edit" will actually delete all empty slides & merge duplicate slides into the same group (it requires 1 change to update)! - But I'll add a feature to do that automatically do that on creation.
2. You can merge slides (as requested here https://github.com/ChurchApps/FreeShow/issues/580) by selecting the slides you want to merge, right click "Format">"Merge". (I notice some bugs with this now that I will try to fix) - I think it also makes sense to merge in the order the slides are selected in (from first to last).

Using the "Text edit" allows for super simple merging and splitting of any slides like you want to!
![bilde](https://github.com/user-attachments/assets/357ce70c-c5cf-49f0-99d2-c3868231e345)

**Comment by joelgraff** [comment]
Ok, more stuff I had no idea was there!  Text Edit certainly does the trick, but it wasn't really "discoverable" unless you've already used right-click / Text view for another purpose. I did discover Text Edit it at one point some time ago, but felt I'd likely have little need for it.  Seems like a great fix to just include that as part of the import process.

As for merge - that looks great!  I'm starting to wonder why I didn't see it before.  Maybe I haven't looked since I updated last week.
